tests/Transports.Subscriptions.WebSockets.Tests/Transports.Subscriptions.WebSockets.Tests.csproj

```diff
@@ -2,17 +2,8 @@
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1;netcoreapp3.0;netcoreapp2.2;net471</TargetFrameworks>
-    <IsPackable>false</IsPackable>
-    <AssemblyName>GraphQL.Server.Transports.WebSockets.Tests</AssemblyName>
-    <RootNamespace>GraphQL.Server.Transports.WebSockets.Tests</RootNamespace>
```

Comment:
Same here. Suggest you keep `RootNamespace`.

Reply:
Done by using naming  template in Directory.Build.props